WordPress/Classes/ViewRelated/Stats/Shared Views/Date Chooser/SiteStatsTableHeaderView.swift

```diff
@@ -247,6 +250,12 @@ private extension SiteStatsTableHeaderView {
     }
 
     func updateButtonStates() {
+        guard !isRunningGhostAnimation else {
+            forwardButton.isEnabled = false
+            backButton.isEnabled = false
+            return
+        }
+
         guard let date = date, let period = period else {
             forwardButton.isEnabled = false
             backButton.isEnabled = false
```

Comment:
maybe we can merge these two `guard` statements and use `isRunningAnimation` inside `updateArrowStates()` accordingly, if needed?

Reply:
As discussed privately we're leaving this as-is for now.  We can evaluate improvements moving forward.